### PR TITLE
Create scriptExecutionErrors.lcdoc

### PR DESCRIPTION
**Pull request by asayd**
This property is undocumented. Creating dictionary entry.

### CODE REVIEW

**Comment by asayd** [comment]
@livecodeali @montegoulding Never created a .lcdoc from whole cloth before. Hope I did this right.

File: docs/dictionary/property/scriptExecutionErrors.lcdoc [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+Type: property
+
+Syntax: put the scriptExecutionErrors
+
+Summary:
+Reports a list of all possible LiveCode script execution errors.
+
+Introduced: 1.0
+
+OS: mac, windows, linux, html5
```
Comment (montegoulding) [inline docs/dictionary/property/scriptExecutionErrors.lcdoc]: @asayd is this actually available in html5? See comment below

File: docs/dictionary/property/scriptExecutionErrors.lcdoc [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+such that the variable tErr would contain something like this:
+
+    573,4,13,in
+
+This would indicate that the error description is found on line 573 of 
+the <scriptExecutionErrors>, and that the error occured on line 4 of 
+the script, on character 13, the token 'in'.
+
+The <scriptExecutionErrors> property is read-only and cannot be set.
+
```
Comment (montegoulding) [inline docs/dictionary/property/scriptExecutionErrors.lcdoc:58]: @asayd you need a NOTE section here to tell people this property returns empty in standalones. It returns the list only in the IDE and server engines.

Comment (asayd) [reply docs/dictionary/property/scriptExecutionErrors.lcdoc:58]: Done.

File: docs/notes/bugfix-18147.md [hunk 3]
```diff
@@ -0,0 +1 @@
+The scriptExecutionErrors property not listed in dictionary
```
Comment (peter-b) [inline docs/notes/bugfix-18147.md]: @asayd This needs a `#` at the start of the line, please!

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 80917fd

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 80917fde18b85a6b79f93ed6583f1282d5ed423a

**Comment by livecode-vulcan** [comment]
😎 test success 80917fde18b85a6b79f93ed6583f1282d5ed423a
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/465)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/470)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/465)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/465)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/471)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/465)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/465)

**Pull request merged**